### PR TITLE
update the link colors on darkmode

### PR DESCRIPTION
**Pull request by zesuma**
## What are you changing in this pull request and why?

https://app.asana.com/0/1200099998847559/1204216689849797/f

Description of the Issue
Review pages and for any links on backgrounds with the color of #262a38 ($neutral100), change the hover color of those links to #1FA4A3 ($teal30) so that they have appropriate contrast per [WebAIM](https://webaim.org/) requirements.

Links
Use the WebAIM web extension to check contrast
https://wave.webaim.org/extension/

NOTE: I limited the brightness on the links a bit since this was causing unanticipated issues on images washing out. I updated the link color and link hover color to #1FA4A3 on dark mode. 

Preview:
https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/docs/introduction

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/src/css/custom.css [hunk 1]
```diff
@@ -668,6 +679,14 @@ i.theme-doc-sidebar-item-category.theme-doc-sidebar-item-category-level-2.menu__
   color: var(--ifm-color-gray-900);
 }
 
+.alert--secondary,
+.alert--secondary a,
+.alert--secondary svg {
+  --ifm-alert-background-color: #474748;
+  color: white !important;
+  fill: white !important;
+}
```
Comment (JKarlavige) [inline website/src/css/custom.css:688]: Do you know of a page where this is used for me to checkout?

**Review by JKarlavige — CHANGES_REQUESTED** [review]
The color adjustments look great in dark mode! Would it be possible to make the same edits for links within the blog? https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/blog

Right now, the blog links go darker on hover and become hard to read in dark mode.

File: website/src/css/custom.css (hunk 1 above)
Comment (zesuma) [reply website/src/css/custom.css:688]: Here's one with the alert--secondary which the the note alert:
https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/docs/cloud/connect-data-platform/connect-apache-spark

Comment (zesuma) [reply website/src/css/custom.css:688]: Good catch @JKarlavige. I've added style updates for the blog hrefs.

**Review by JKarlavige — APPROVED** [review]
lgtm!! :shipit:

**Review by breezyfasano — CHANGES_REQUESTED** [review]
Hi Trever! Woo this is a big one, thanks for submitting this. I love these accessibility updates but changes like these are tricky cuz they touch every page!

I noticed that on [the Materializations page](https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/docs/build/materializations), at the very bottom, the Discourse widget CTA is now washed out and does not meet contrast guidelines.
Screenshot 2023-10-09 at 10 34 38 AM

After checking a fair amount of pages, that was the only issue I could find! Nice work.

**Comment by zesuma** [comment]
> Hi Trevor! Woo this is a big one, thanks for submitting this. I love these accessibility updates but changes like these are tricky cuz they touch every page!
> 
> I noticed that on [the Materializations page](https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/docs/build/materializations), at the very bottom, the Discourse widget CTA is now washed out and does not meet contrast guidelines. Screenshot 2023-10-09 at 10 34 38 AM
> 
> After checking a fair amount of pages, that was the only issue I could find! Nice work.

great catch @breezyfasano ! Yeah it was very tricky! You fix one thing and find you've messed up another thing lol

**Comment by zesuma** [comment]
> > Hi Trevor! Woo this is a big one, thanks for submitting this. I love these accessibility updates but changes like these are tricky cuz they touch every page!
> > I noticed that on [the Materializations page](https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/docs/build/materializations), at the very bottom, the Discourse widget CTA is now washed out and does not meet contrast guidelines. Screenshot 2023-10-09 at 10 34 38 AM
> > After checking a fair amount of pages, that was the only issue I could find! Nice work.
> 
> great catch @breezyfasano ! Yeah it was very tricky! You fix one thing and find you've messed up another thing lol

should be fixed now: https://docs-getdbt-com-git-fix-insufficient-contrast-o-3250d0-dbt-labs.vercel.app/docs/build/materializations

**Review by breezyfasano — APPROVED** [review]
lgtm!! Thanks for fixing Trever @zesuma ! I'll go ahead and merge now.

**Pull request merged**